<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test2Dialog</class>
 <widget class="QDialog" name="Test2Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Інформаційно-технологічний тест</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1053</width>
        <height>887</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>5. Чи володієте Ви навичками пошуку в Інтернет?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb51">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb52">
            <property name="text">
             <string>Контекстний пошук за ключовим словом</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb53">
            <property name="text">
             <string>Контекстний пошук із застосуванням логічних умов та пошук файлів</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb54">
            <property name="text">
             <string>Володію усіма можливими видами пошуку у Інтернет</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>1. Чи володієте Ви уміннями роботи з текстовим редактором (стильове форматування, автозміст, таблиці, рисунки)?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb11">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb12">
            <property name="text">
             <string>Тільки з набору тексту</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb13">
            <property name="text">
             <string>Cтильове форматування, автозміст, таблиці, рисунки</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb14">
            <property name="text">
             <string>Володію більшістю функцій текстового редактору</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>3. Чи володієте Ви уміннями щодо розроблення презентацій (наприклад, у Microsoft PowerPoint)?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb31">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rb34">
            <property name="text">
             <string>Володію більшістю функцій</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb33">
            <property name="text">
             <string>Подання тексту, рисунків, діаграм, анімації</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb32">
            <property name="text">
             <string>Тільки подання тексту</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>rb31</zorder>
         <zorder>rb32</zorder>
         <zorder>rb33</zorder>
         <zorder>rb34</zorder>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>6. Чи володієте Ви навичками подання інформації із застосуванням гіперексту (HTML)?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb61">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb62">
            <property name="text">
             <string>Розробка і редагування веб-сторінки</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb63">
            <property name="text">
             <string>Розробка веб-сайту з графічним наповнюванням</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb64">
            <property name="text">
             <string>Основи розробки веб-сайтів із застосуванням різноманітних вебконструкторів й технологій PHP, Java</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>7. Чи володієте Ви навичками збору й обробки інформації із застосуванням спеціалізованого (комп’ютерного) програмного забезпечення?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb71">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb72">
            <property name="text">
             <string>Володію автоматизованими системами бронювання (пошук, підбір, он-лайн бронювання та купівля залізничних, автобусних й авіаквитків)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb73">
            <property name="text">
             <string>Володію автоматизованими системами бронювання (пошук, підбір, он-лайн бронювання номеру в готелі)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb74">
            <property name="text">
             <string>Володію автоматизованими системами бронювання (пошук туру, підбір, оформлення, он-лайн бронювання, збереження та відправка замовлення туроператору, відстеження стану замовлення)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>4.  Чи володієте Ви уміннями роботи з програмним забезпеченням Інтернет-коммунікації?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="4" column="0">
           <widget class="QRadioButton" name="rb43">
            <property name="text">
             <string>ICQ, Skype, IRC, Viber, Watsapp</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb42">
            <property name="text">
             <string>Електронна пошта</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb41">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="rb44">
            <property name="text">
             <string>Володію більшістю програмного забезпечення для Інтернет-комунікації</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>2. Чи володієте Ви уміннями роботи з електронними таблицями (наприклад, Microsoft)?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="5" column="0">
           <widget class="QRadioButton" name="rb24">
            <property name="text">
             <string>Володію більшістю функцій електронних таблиць</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb22">
            <property name="text">
             <string>Побудова графіків та діаграм</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb23">
            <property name="text">
             <string>Обробка даних з маркетингового аналізу та бухгалтерського обліку</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb21">
            <property name="text">
             <string>Не володію</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Test2Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Test2Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
